code/rewriter.go

```diff
@@ -453,15 +418,34 @@ func (r *Rewriter) rewriteStmts(stmts []ast.Stmt) error {
 				return err
 			}
 
+		case *ast.SendStmt:
+			err := r.rewriteExprs([]ast.Expr{v.Chan, v.Value})
+			if err != nil {
+				return err
+			}
+
+		case *ast.LabeledStmt:
+			stmts := []ast.Stmt{v.Stmt}
+			err := r.rewriteStmts(stmts)
+			if err != nil {
+				return err
+			}
+			v.Stmt = stmts[0]
```

Comment:
is this assignment necessary ?

Reply:
Yes, rewriteStmts will replace `stmts` element that does not affect original `v.Stmt`